src/app/ubs-admin/components/ubs-admin-go-back-modal/ubs-admin-go-back-modal.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, OnInit } from '@angular/core';
+import { MatDialogRef } from '@angular/material/dialog';
+
+@Component({
+  selector: 'app-ubs-admin-go-back-modal',
+  templateUrl: './ubs-admin-go-back-modal.component.html',
+  styleUrls: ['./ubs-admin-go-back-modal.component.scss']
+})
+export class UbsAdminGoBackModalComponent implements OnInit {
+  constructor(private dialogRef: MatDialogRef<UbsAdminGoBackModalComponent>) {}
+
+  ngOnInit() {}
```

Comment:
ngOnInit is empty.
![image](https://user-images.githubusercontent.com/43093994/126048120-e223cf88-fe57-4959-afb3-f07eed45a9a4.png)

Reply:
done